diff -Nurp a/configure.in b/configure.in
--- a/configure.in	2013-03-25 16:49:33.764797537 +0100
+++ b/configure.in	2013-03-25 16:54:07.377479592 +0100
@@ -906,6 +906,13 @@ case "${GLIBC_VERSION}" in
 	DEFAULT_SUPP="glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
 	DEFAULT_SUPP="glibc-2.X-drd.supp ${DEFAULT_SUPP}"
 	;;
+     2.17)
+        AC_MSG_RESULT(2.17 family)
+        AC_DEFINE([GLIBC_2_17], 1, [Define to 1 if you're using glibc 2.17.x])
+        DEFAULT_SUPP="glibc-2.X.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="glibc-2.X-drd.supp ${DEFAULT_SUPP}"
+        ;;
      darwin)
 	AC_MSG_RESULT(Darwin)
 	AC_DEFINE([DARWIN_LIBC], 1, [Define to 1 if you're using Darwin])
@@ -919,7 +926,7 @@ case "${GLIBC_VERSION}" in
 
      *)
 	AC_MSG_RESULT([unsupported version ${GLIBC_VERSION}])
-	AC_MSG_ERROR([Valgrind requires glibc version 2.2 - 2.16])
+	AC_MSG_ERROR([Valgrind requires glibc version 2.2 - 2.17])
 	AC_MSG_ERROR([or Darwin libc])
 	;;
 esac
