diff -purN libxslt-1.1.26.orig/configure libxslt-1.1.26/configure
--- libxslt-1.1.26.orig/configure	2009-12-16 16:50:48.000000000 +0100
+++ libxslt-1.1.26/configure	2009-12-16 17:07:50.000000000 +0100
@@ -15860,32 +15860,9 @@ $as_echo "$as_me: error: libxml source d
 	fi
 fi
 
-
-if ${XML_CONFIG} --libs print > /dev/null 2>&1
-then
-	XMLVERS=`$XML_CONFIG --version`
-	if test `echo $XMLVERS | sed -e 's/libxml //' | awk 'BEGIN { FS = "."; } { printf "%d", ($1 * 1000 + $2) * 1000 + $3;}'` -ge `echo $LIBXML_REQUIRED_VERSION | sed -e 's/libxml //' | awk 'BEGIN { FS = "."; } { printf "%d", ($1 * 1000 + $2) * 1000 + $3;}'`
-	then
-		LIBXML_LIBS="$LIBXML_LIBS `$XML_CONFIG --libs`"
-		LIBXML_CFLAGS="$LIBXML_CFLAGS `$XML_CONFIG --cflags`"
-		{ $as_echo "$as_me:$LINENO: result: $XMLVERS found" >&5
-$as_echo "$XMLVERS found" >&6; }
-	else
-		{ { $as_echo "$as_me:$LINENO: error: Version $XMLVERS found. You need at least libxml2 $LIBXML_REQUIRED_VERSION for this version of libxslt" >&5
-$as_echo "$as_me: error: Version $XMLVERS found. You need at least libxml2 $LIBXML_REQUIRED_VERSION for this version of libxslt" >&2;}
-   { (exit 1); exit 1; }; }
-	fi
-else
-	{ { $as_echo "$as_me:$LINENO: error: Could not find libxml2 anywhere, check ftp://xmlsoft.org/." >&5
-$as_echo "$as_me: error: Could not find libxml2 anywhere, check ftp://xmlsoft.org/." >&2;}
-   { (exit 1); exit 1; }; }
-fi
-
-
-
-
-
-
+XMLVERS=`$XML_CONFIG --version`
+LIBXML_LIBS="-L${SRC_DIR} -lxml2 -lz -lm"
+LIBXML_CFLAGS="-I${SRC_DIR}/include -I${SRC_DIR}/include/libxml2"
 
 # Check whether --with-plugins was given.
 if test "${with_plugins+set}" = set; then
