diff -Nur lighttpd-1.4.31.orig/src/server.c lighttpd-1.4.31/src/server.c
--- lighttpd-1.4.31.orig/src/server.c	2012-04-28 17:39:38.000000000 +0200
+++ lighttpd-1.4.31/src/server.c	2012-09-21 14:14:06.391779237 +0200
@@ -783,12 +783,6 @@
 						"can't find username", srv->srvconf.username);
 				return -1;
 			}
-
-			if (pwd->pw_uid == 0) {
-				log_error_write(srv, __FILE__, __LINE__, "s",
-						"I will not set uid to 0\n");
-				return -1;
-			}
 		}
 
 		if (srv->srvconf.groupname->used) {
@@ -797,11 +791,6 @@
 					"can't find groupname", srv->srvconf.groupname);
 				return -1;
 			}
-			if (grp->gr_gid == 0) {
-				log_error_write(srv, __FILE__, __LINE__, "s",
-						"I will not set gid to 0\n");
-				return -1;
-			}
 		}
 #endif
 		/* we need root-perms for port < 1024 */
